cypress/e2e/RoundTrips/SimpleProject.feature

```diff
@@ -72,10 +73,10 @@ Feature: Test roundtrip of the application: project creation
     And   I set context field 'projectRenamed' with 'renamedProject'
     And   I set context field 'modelRenamed' with 'newModelTest'
     And   I visit the '/'
-    And   I wait until the application is loaded
+    Then  I wait until the application is loaded
```

Comment:
keep And here because it's "Given I wait until the application is loaded"

Reply:
oh true, it was a test, forgot to remove it

Reply:
done